vectorbender.py

```diff
@@ -139,13 +148,13 @@ def run(self):
 
         # Loading the delaunay
         restrictToSelection = self.dlg.restrictBox_pairsLayer.isChecked()
-        if transType==4:
```

Comment:
yes you are right, but: why to ask to the user wich kind of transformation to use?
may be we have to explain very well, the differences of these two methods.
as I said, i'm not understanding the bender one...